doc/guides/west/basics.rst

```diff
@@ -1,19 +1,16 @@
-.. _west-multi-repo:
+.. _west-basics:
 
 Basics
 ######
 
-This page introduces west's basic concepts and built-in commands along with
-references to further reading.
-
-.. _west-workspace:
-
-Introduction
-************
+This page introduces west's basic concepts and provides references to further
+reading.
 
 West's built-in commands allow you to work with *projects* (Git
-repositories) under a common *workspace* directory. You can create a
-workspace using the :ref:`west-init` command.
+repositories) under a common *workspace* directory.
+
+Example workspace
+*****************
```

Comment:
You do some very nice cleanups in this PR, so maybe you could cleanup this as well.
Comment belong to those lines that are not touched in the PR: https://github.com/zephyrproject-rtos/zephyr/blob/38bac6da34ac47e6e79e919e617a0ce9b8367310/doc/guides/west/basics.rst#L18-L30

At least I find it this hard to read.

Maybe something like:
```
zephyrproject/                 # west topdir
├── .west/                     # marks the location of the topdir
│   └── config                 # per-workspace local configuration file
│
│   # Below, the manifest repository, never modified by west after creation
├── zephyr/                    # .git/ repo
│   ├── west.yml               # manifest file
│   └── [... other files ...]
│
│   # Below, projects updated by west
├── modules/
│   └── lib/
│       └── tinycbor/          # .git/ project
├── net-tools/                 # .git/ project
└── [ ... other projects ...]
```

Reply:
> At least I find this hard to read.

Good idea. I added another patch.